examples/discovery-search-app/cypress/integration/basic/spec.ts

```diff
@@ -73,3 +74,62 @@ describe('Basic search', () => {
     });
   });
 });
+
+describe('Basic search errors', () => {
+  beforeEach(() => {
+    // start the cypress server
+    cy.server();
+
+    // Sets up and handles the collections, component settings, and initial query requests that run on page-load
+    cy.fixture('collections/collections').as('collectionsJSON');
+    cy.route('GET', '**/collections?version=2019-01-01', '@collectionsJSON').as('getCollections');
+    cy.fixture('component_settings/componentSettings').as('componentSettingsJSON');
+    cy.route('GET', '**/component_settings?version=2019-01-01', '@componentSettingsJSON').as(
+      'getComponentSettings'
+    );
+
+    // Set up/override routes & fixtures that are specific to this file
+    cy.fixture('query/query').as('queryJSON');
+    cy.fixture('query/error500.json').as('error500JSON');
+  });
+
+  describe('When loading query returns an error', () => {
+    it('does something', () => {
+      // query request on initial load -- throws error
+      cy.route({
+        method: 'POST',
+        url: '**/query?version=2019-01-01',
+        response: '@error500JSON',
+        status: 500
+      }).as('postQueryError');
```

Comment:
This fails because the call to `searchClient.query` in `DiscoverySearch::handleFetchAggregations()` isn't wrapped in a try/catch.

Compare to the invocation of `searchClientMethod.call` in `useDataApi::fetchData()`, which wraps in try/catch.